patches/nginx-1.17.4-intercept_error_log.patch

```diff
@@ -3,7 +3,7 @@ index c51b7ff..4c335b9 100644
 --- a/src/core/ngx_cycle.h
 +++ b/src/core/ngx_cycle.h
 @@ -22,9 +22,14 @@
- #define NGX_DEBUG_POINTS_ABORT  2
+ #define HAVE_PRIVILEGED_PROCESS_PATCH   1
```

Comment:
What is this change?

Reply:
The `src/core/ngx_cycle.h` was patched by the `privileged-agent-process` before

https://github.com/openresty/openresty/blob/7508c1852265bd04fdb2dfd64989d4c490440f1a/patches/nginx-1.17.4-privileged_agent_process.patch#L25-L34